modules/SiteHeader/SiteHeaderModule.php

```diff
@@ -23,10 +23,16 @@ public function __construct(ContentPartialService $contentService)
     public function init(): void
     {
         $this->contentService->addTemplateArea('header');
+
+        $this->enqueueFrontendAssets(function (ModuleAssets $assets) {
+            $handle = 'site-header';
+            $assets->enqueueStyle($handle, 'main.css');
+            $assets->enqueueScript($handle, 'main.mjs', ['wp-hooks']);
```

Comment:
This doesn't need `wp-hooks` as a dependency, but it does need `sitchco/ui-framework`.